File: app/src/main/java/dev/ragnarok/fenrir/link/internal/OtherLink.java
package dev.ragnarok.fenrir.link.internal;

public class OtherLink extends AbsInternalLink {
    public final String Link;

    public OtherLink(int start, int end, String link, String name) {
        this.start = start;
        this.end = end;
        targetLine = name;
        Link = link;
    }
}

File: app/src/main/java/dev/ragnarok/fenrir/link/internal/OtherLink.kt
package dev.ragnarok.fenrir.link.internal

class OtherLink(start: Int, end: Int, link: String, name: String) : AbsInternalLink() {
    @JvmField
    val Link: String

    init {
        this.start = start
        this.end = end
        targetLine = name
        Link = link
    }
}